packages/geolocation/package.json

```diff
@@ -36,52 +36,47 @@
     "require": "./dist/index.cjs"
   },
   "scripts": {
-    "start": "vite -r ./dev/ -c ./dev/vite.config.ts",
+    "start": "vite serve dev",
+    "dev": "vite serve dev",
     "build": "tsup",
-    "test": "echo disabled defunct jest test suite"
+    "test": "uvu -b -r solid-register"
   },
   "keywords": [
     "geolocation",
     "location",
     "coordinates",
     "primitives"
   ],
+  "testEnvironment": "jsdom",
   "devDependencies": {
-    "@babel/preset-env": "^7.15.8",
+    "@babel/preset-env": "^7.18.2",
     "@types/jest": "^27.4.0",
-    "babel-preset-solid": "^1.1.5",
-    "jest": "^27.2.5",
-    "prettier": "^2.7.1",
+    "@babel/preset-env": "^7.18.2",
+    "@types/jest": "^27.4.0",
+    "@types/leaflet": "^1.7.11",
+    "babel-preset-solid": "^1.4.4",
+    "jsdom": "^19.0.0",
+    "leaflet": "^1.8.0",
+    "mapbox-gl": "^2.9.0",
+    "prettier": "^2.0.5",
+    "solid-map-gl": "^1.3.3",
+    "solid-register": "^0.2.5",
     "solid-testing-library": "^0.2.0",
     "ts-jest": "^27.1.2",
     "tslib": "^2.0.1",
-    "tsup": "^6.1.2",
+    "tsup": "^5.10.1",
     "typescript": "^4.7.4",
-    "wait-for-expect": "^3.0.2"
+    "unocss": "^0.39.1",
+    "uvu": "^0.5.3",
+    "vite": "^2.9.12",
+    "vite-plugin-mkcert": "^1.7.2",
+    "vite-plugin-solid": "^2.2.6",
+    "vitest": "^0.15.1"
```

Comment:
Some dev deps here are duplicated, and some are not needed at all.
vitest, babel-preset-solid, @types/jest and @babel/preset-env could be safely removed I think

Reply:
Thanks for pointing that out! :-)